src/ulong_extras/test/t-mulmod_shoup.c

```diff
@@ -15,30 +16,29 @@
 
 TEST_FUNCTION_START(n_mulmod_shoup, state)
 {
-    int i, result;
-
-    for (i = 0; i < 10000 * flint_test_multiplier(); i++)
+    for (ulong i = 0; i < 10000 * flint_test_multiplier(); i++)
```

Comment:
If I'm allowed to be picky, I would like to keep
```c
int i, result;
```
as before. I think it is important that we try to stick to a very consistent style, not only for people coming into the codebase, but also for purposes of parsing like through `sed` or whatever.

Reply:
Ah, ok, sure, for these parts that are identical in all test/profile files it is convenient to keep them consistent. I will change it back.